### DISCUSSION TITLE
Need assistance in running DESeq2

### DISCUSSION THREAD

**Discussion by Rajeshwarira** [GuyTeichman/RNAlysis#27] [Q&A]
### Describe the bug

Hello Guy,
I tried to run DESeq2 and I'm getting the following error
Traceback (most recent call last):

  File "rnalysis\gui\gui.py", line 3697, in finish_generic_job

  File "rnalysis\gui\gui_widgets.py", line 329, in run

  File "rnalysis\filtering.py", line 2874, in differential_expression_deseq2

  File "rnalysis\utils\differential_expression.py", line 130, in run_deseq2_analysis

  File "rnalysis\utils\io.py", line 1397, in run_r_script

AssertionError: R script failed to execute (return code 1).

### Steps To Reproduce

I need help to get over the error mentioned above.
Please do help me with this.

### How did you install RNAlysis?

Stand-alone app

### RNAlysis version

3.7

### Operating system

Windows

**Comment by GuyTeichman** [comment]
Hi,
To be able to figure out why your differential expression analysis failed, you will need to provide more details: what design matrix were you using? What was printed to your log box? Do you have R installed on your computer?

**Comment by Rajeshwarira** [comment]
Hello Guy,
     I'm sorry for the late reply.
     Yes, I have R(4.3) installed
      I have attached the design matrix and log box(photo) with this comment.
      Please look into it and help me with this.
![log box ](https://github.com/GuyTeichman/RNAlysis/assets/128403613/1726e01f-8bd1-435f-bfe9-bbba63ed157d)

      
      
[design matrix 1.1.csv](https://github.com/GuyTeichman/RNAlysis/files/11767359/design.matrix.1.1.csv)
[Q.1.1 ctrl vs cases.csv](https://github.com/GuyTeichman/RNAlysis/files/11767361/Q.1.1.ctrl.vs.cases.csv)

**Comment by GuyTeichman** [comment]
Hi,
I suspect that your analysis is failing for multiple reasons:
1. because your design matrix is not full rank
2. the names of the factors in your design matrix contain spaces
please try running again with RNAlysis version 3.9.1 after correcting these, and let me know if you run into the same error message.
